Meta/CMake/processor-count.cmake

```diff
@@ -1,3 +1,5 @@
 include(ProcessorCount)
 ProcessorCount(N)
-message("${N}")
+# Executing echo here allows us to print to the standard output,
+# to separate the processor count from potential errors
+execute_process(COMMAND ${CMAKE_COMMAND} -E echo "${N}")
```

Comment:
Does `${CMAKE_COMMAND}` not require any quotes around it? It looks like most other places use it as `"${CMAKE_COMMAND}"` instead.

Reply:
some places use quotes, some other do not (e.g. Kernel/CMakeLists.txt, Userland/Libraries/LibC/CMakeLists.txt)
according to https://stackoverflow.com/a/35853080, CMake does the same variable splitting as shells, which means that it might fail if cmake's path has spaces in it
should i open another PR to fix other ${CMAKE_COMMAND} ocurrences?

Reply:
> should i open another PR to fix other ${CMAKE_COMMAND} ocurrences?

If you want to, you may, obviously. :^)

The fix for the line that was added here should obviously be squashed into this commit.

Reply:
#15111 15111